charts/amoro/values.yaml

```diff
@@ -0,0 +1,162 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+# AMS server numbers
+replicaCount: 1
```

Comment:
think it's more appropriate to specify this configuration in the HA section.

Reply:
The first initial release did not support multi-copy deployment, I suggest that this part currently needs to mask the HA multi-copy configuration, which would be outside the scope of this issue if code support was required.I will temporarily block this option and default to 1 cc @baiyangtx